modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SlimFrameworkServerCodegen.java

```diff
@@ -0,0 +1,195 @@
+import io.swagger.codegen.DefaultCodegen;
+import io.swagger.codegen.SupportingFile;
+import io.swagger.models.properties.ArrayProperty;
+import io.swagger.models.properties.MapProperty;
+import io.swagger.models.properties.Property;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+
+public class SlimFrameworkServerCodegen extends DefaultCodegen implements CodegenConfig {
+    protected String invokerPackage;
+    protected String groupId = "io.swagger";
+    protected String artifactId = "swagger-server";
+    protected String artifactVersion = "1.0.0";
+
+    public SlimFrameworkServerCodegen() {
+        super();
+
+        invokerPackage = camelize("SwaggerServer");
+
+        String packagePath = "SwaggerServer";
+
+        modelPackage = packagePath + "\\lib\\Models";
+        apiPackage = packagePath + "\\lib";
+        outputFolder = "generated-code" + File.separator + "slim";
+        modelTemplateFiles.put("model.mustache", ".php");
+
+        // no api files
+        apiTemplateFiles.clear();
+
+        embeddedTemplateDir = templateDir = "slim";
+
+        reservedWords = new HashSet<String>(
+                Arrays.asList(
+                        "__halt_compiler", "abstract", "and", "array", "as", "break", "callable", "case", "catch", "class", "clone", "const", "continue", "declare", "default", "die", "do", "echo", "else", "elseif", "empty", "enddeclare", "endfor", "endforeach", "endif", "endswitch", "endwhile", "eval", "exit", "extends", "final", "for", "foreach", "function", "global", "goto", "if", "implements", "include", "include_once", "instanceof", "insteadof", "interface", "isset", "list", "namespace", "new", "or", "print", "private", "protected", "public", "require", "require_once", "return", "static", "switch", "throw", "trait", "try", "unset", "use", "var", "while", "xor")
+        );
+
+        additionalProperties.put(CodegenConstants.INVOKER_PACKAGE, invokerPackage);
+        additionalProperties.put(CodegenConstants.GROUP_ID, groupId);
+        additionalProperties.put(CodegenConstants.ARTIFACT_ID, artifactId);
+        additionalProperties.put(CodegenConstants.ARTIFACT_VERSION, artifactVersion);
+
+        // ref: http://php.net/manual/en/language.types.intro.php
+        languageSpecificPrimitives = new HashSet<String>(
+                Arrays.asList(
+                        "boolean",
+                        "int",
+                        "integer",
+                        "double",
+                        "float",
+                        "string",
+                        "object",
+                        "DateTime",
+                        "mixed",
+                        "number")
+        );
+
+        instantiationTypes.put("array", "array");
+        instantiationTypes.put("map", "map");
+
+        // ref: https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#data-types
+        typeMapping = new HashMap<String, String>();
```

Comment:
In https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/PhpClientCodegen.java#L92 is a second DateTime with capital D and T, but none of them fit the case sensitive dateTime specified in the swagger-spec. I'm not quite sure whether this is case sensitive, but I guess not. I've taken out the second DateTime.